src/user/introduction.rst

```diff
@@ -47,9 +47,11 @@ conda-forge is a community effort that tackles these issues:
 How can I install packages from conda-forge?
 --------------------------------------------
 
-Using conda-forge is as easy! Make sure you have at least ``conda >=4.6``,
-then you just have to register the conda-forge channel as a package source for conda,
-and activate the ``strict`` channel priority (``strict`` will be activated by default in conda 5.0).
+Using conda-forge is easy!
+
+ - Make sure you have ``conda >=4.9``.
+ - Register the conda-forge channel as a package source for conda.
```

Comment:
This text should probably be restated as "Add ``conda-forge`` as the first channel in your ``.condarc``" or something similar.

Reply:
Hey, @beckermr !
I'd need help understanding this. 
Do you mean we should be more clear when we say " Register the conda-forge channel as a package source for conda." ?

Reply:
Okay, I got it now.

Reply:
I think the better approach would be something similar as what Matt said, containing some information of condarc file for instance (just the `conda config` command would be enough) and the `conda config --add channels conda-forge` command to add conda-forge into the channels list

Reply:
Something in these lines "... Add conda-forge as the first channel in your .condarc.  The .condarc file is not included by default, but it is automatically created in your home directory the first time you run the conda config command.  To add conda-forge to your chanbels list just type: `conda config --add channels conda-forge` or modify these lines ... in you condarc config file."

Reply:
@viniciusdc 
As long as we can do without it, I wanted to avoid mentioning '.condarc file' here. 
This section starts with 'Using conda-forge is easy!', so I think, it would be best if we kept the configuration guide as simple as possible. 

But, adding a link to conda docs where the '.condarc' file is described could be a good idea.

Reply:
Better still, we could add this link instead :
https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html

It explains channel priority and it's effects.

Reply:
Another thing,
Do you think it's a good idea to add the corresponding command immediately after the step. 

Like this: 

 - Make sure you have ``conda >=4.9``.
 - Add ``conda-forge`` as the highest priority channel. 
    
     `conda config --add channels conda-forge`
 
 - Activate the ``strict`` channel priority (``strict`` will be activated by default in conda 5.0).
     
     `conda config --set channel_priority strict`

Reply:
Yes! This is great. We should put a conda update command there for the first bullet point.

Reply:
@beckermr 
Yes, right!
Matt, will you please take a look at #1280 ?